firestore/schedule-export/index.js

```diff
@@ -0,0 +1,26 @@
+const firestore = require('@google-cloud/firestore');
+const client = new firestore.v1.FirestoreAdminClient();
+// Replace BUCKET_NAME
+const bucket = 'gs://BUCKET_NAME';
+
+exports.scheduledFirestoreExport = (event, context) => {
+
+const databaseName =
```

Comment:
Nit: can we indent this function body?

Related:  make sure this passes `eslint`, without any `WARNING`s or `ERROR`s.

Reply:
Done. Ran npm run lint -- --fix